identity/src/main/java/com/android/identity/AccessControlProfile.java

```diff
@@ -38,6 +38,9 @@ AccessControlProfileId getAccessControlProfileId() {
         return mAccessControlProfileId;
     }
 
+    /**
+     * Returns the authentication timeout, in milliseconds.
+     */
     long getUserAuthenticationTimeout() {
         return mUserAuthenticationTimeout;
```

Comment:
Would you mind also renaming the field to `mUserAuthenticationTimeoutMillis` just so units are obvious everywhere internally, too?

Reply:
Done (I've git push --force'ed a new version of the commit).